pkg/controller/migration/controller.go

```diff
@@ -249,19 +249,21 @@ func (r *Reconciler) setOwnerReference(migration *api.Migration) (*api.Migration
 		},
 		plan,
 	)
-
 	if err != nil {
-		return nil, err
+		if k8serr.IsNotFound(err) {
```

Comment:
```suggestion
                // ... some reason we ignore this error ...
		if k8serr.IsNotFound(err) {
```

much nicer fix then adding the check before calling this method ! nice catch, making this method more flexible !